modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/index.js

```diff
@@ -116,15 +116,22 @@ export function getValueDetailsFromItem(item, fieldName) {
 			if (property === 'LANG') {
 				const languageId = Liferay.ThemeDisplay.getLanguageId();
 				const BCP47LanguageId = Liferay.ThemeDisplay.getBCP47LanguageId();
+				const defaultInstanceLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
 
 				if (navigatedValue[languageId]) {
 					formattedProperty = languageId;
 				}
 				else if (navigatedValue[BCP47LanguageId]) {
 					formattedProperty = BCP47LanguageId;
 				}
+				else if (navigatedValue[defaultInstanceLanguageId]) {
+					formattedProperty = defaultInstanceLanguageId;
+				}
+				else if (navigatedValue['en_US']) {
```

Comment:
Hey @thienquach87, Is there a reason why you add this conditional explicit for `en_US` property?